src/MapboxMap/index.jsx

```diff
@@ -245,6 +258,7 @@ MapboxMap.propTypes = {
     maxZoom: PropTypes.number,
     minZoom: PropTypes.number,
     navigationControlPosition: PropTypes.string,
+    navigationType: PropTypes.shape({type: PropTypes.oneOf(navigationMethods), options: PropTypes.shape({speed: PropTypes.number, curve: PropTypes.number})}),
```

Comment:
J'ai l'impression que le fichier n'est pas formaté. Il n'y a pas de hook de pre-commit sur `react-mapbox-wrapper` ?

Reply:
Je ne sais pas trop je ne suis pas encore familière avec ce qu'est un hook de pre-commit 😕

Reply:
C'est cela ?
You can use our `pre-commit` hook for ensuring consistent format and linting before committing by running command:

```bash
./tools/install_hooks.sh
```

Reply:
C'est ça ! 😄
Normalement si tu les installes ils s'occupent de vérifier plein de chose, notamment que les fichier sont bien formatés (et il me semble que si ce n'est pas le cas il les format tout seul).